### PR TITLE
Feat/deliverable1

### PR DESCRIPTION
**Pull request by albertobautista**

### CODE REVIEW

File: src/components/ProductItem/index.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <p className="card-text fs-5">{currencyFormat(price)}</p>
+        </div>
+        <div className="text-center d-grid gap-2 col-12 mx-auto">
+          <button className="btn btn-outline-primary btn-md">
+            Add to cart
+          </button>
+        </div>
+      </div>
+    </div>
+    // <div className="ibox">
```
Comment (YeipiLozano) [inline src/components/ProductItem/index.jsx]: try to push code without comments

File: src/components/ProductItem/index.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <img
+          src={images[0]}
+          className="card-img-top"
+          alt={name}
+          style={{ width: "100%", height: "15vw", objectFit: "contain" }}
+        />
+      </div>
+      <div className="card-body">
+        <h5 className="card-title fw-bold" style={{ minHeight: "80px" }}>
+          {name.substring(0, 80) + "..."}
```
Comment (YeipiLozano) [inline src/components/ProductItem/index.jsx]: I see what you're doing here, maybe a better approach could be managing it with CSS, try looking up how to truncate text or use line-clamp to do it

File: src/utils/currency.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const formatDecimalNumber = (number) =>
```
Comment (YeipiLozano) [inline src/utils/currency.js:1]: Excellent touch

**Review by YeipiLozano — CHANGES_REQUESTED** [review]
I see you have some problems with the layout, try adding the margin to the image container, and add max-height:100% to the image itself, maybe that'll work! Super great job
image

File: src/components/ProductItem/index.jsx (hunk 1 above)
Comment (albertobautista) [reply src/components/ProductItem/index.jsx]: I deleted the comments

File: src/components/ProductItem/index.jsx (hunk 2 above)
Comment (albertobautista) [reply src/components/ProductItem/index.jsx]: I fixed it

File: src/utils/currency.js (hunk 3 above)
Comment (albertobautista) [reply src/utils/currency.js:1]: Thanks

**Pull request merged**